### ISSUE TITLE
Folder not Found

### ISSUE THREAD

**Issue by code007-1** [UnitySuperEditor/SuperEditor#8]
In unity 2020.1.1f1, When opening any file window it says, 

AssetDatabase.FindAssets: Folder not found: 'SuperEditor/EditorResources/Fonts'
UnityEditor.AssetDatabase:FindAssets(String, String[])
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:MALGDKADEDLONPFACJAGNOJFCCJHGAELLKBK()
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:LoadStyles(GPIOONJLKBMHJKDDEPBFBBNPIJALEMMEKBBM, Boolean)
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:Initialize()
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:OnEnable(Object)
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK:OnFirstRepaint()
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK:OnGUI()
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)

And show empty file window

**Comment by UnitySuperEditor** [comment]
Are you using Linux？

**Comment by code007-1** [comment]
No, I am using Windows 8.1 and latest version of super editor available in unity asset store

**Comment by UnitySuperEditor** [comment]
Have you moved the SuperEditor folder? If so, just restart the Unity Editor

**Comment by code007-1** [comment]
I tried before moving SuperEditor Folder but it gives error and after moving I restarted unity editor but still getting same error

**Comment by UnitySuperEditor** [comment]
Thank you for your feedback. I will download the Win8 simulator and test it, because it is OK to run in Win10

**Comment by UnitySuperEditor** [comment]
The new version 1.2.2 has been released, It adds an emergency handling mechanism, which is a fixed path search for fonts, You can try it out and see if it works

**Comment by code007-1** [comment]
Still no luck after updating to 1.2.200 version, Now i am getting this error 

AssetDatabase.FindAssets: Folder not found: 'Assets/EditorResources/Fonts'
UnityEditor.AssetDatabase:FindAssets (string,string[])
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:MALGDKADEDLONPFACJAGNOJFCCJHGAELLKBK ()
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:LoadStyles (CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM/GPIOONJLKBMHJKDDEPBFBBNPIJALEMMEKBBM,bool)
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:Initialize ()
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.LBBGPIDBLEGLKGGBIIFGOBJNOJGIILIIEFMM:OnEnable (UnityEngine.Object)
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK:OnFirstRepaint ()
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK:OnGUI ()
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)


As It was Searching for this folder - 'Assets/EditorResources/Fonts' and SuperEditor file structure is 
different like this- 'SuperEditor/EditorResources/Fonts'
So i renamed the "SuperEditor" Folder to "Assets" and now getting these three errors even after restart unity editor

AssetDatabase.FindAssets: Folder not found: 'Assets/EditorResources/Fonts'
UnityEditor.AssetDatabase:FindAssets (string,string[])

AssetDatabase.FindAssets: Folder not found: 'Assets/SuperEditor/EditorResources/Fonts'
UnityEditor.AssetDatabase:FindAssets (string,string[])

AssetDatabase.FindAssets: Folder not found: 'Assets/Plugins/SuperEditor/EditorResources/Fonts'
UnityEditor.AssetDatabase:FindAssets (string,string[])

Reverted renamed folder as before and getting same error as before


I was typing some random words in File window SuperEditor it showed one word on top of other and gave this new error

NullReferenceException: Object reference not set to an instance of an object
CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.CGHBNFMMNFMFJNOPDHLEBKJFABAGBPBNAALK.<AddItemsToMenu>b__97_8 () (at <d15b8b481b644a598084443151f67578>:0)
UnityEditor.GenericMenu.CatchMenu (System.Object userData, System.String[] options, System.Int32 selected) (at <756e9f00f78640ec93f83e08257daaec>:0)

**Comment by UnitySuperEditor** [comment]
public static void Test()
{
		var guids = AssetDatabase.FindAssets("t:Font", new string[] { "Assets/SuperEditor/EditorResources/Fonts" });
		UnityEngine.Debug.Log(guids.Length);
}
Can you enter this method in your project and see what the output is, I guess there is a problem with the internal API of UnityEditor on Linux platforms, because I find that some of the APIs on Ubuntu are very different from Windows.

**Comment by UnitySuperEditor** [comment]
Like this
<img width="621" alt="屏幕快照 2021-05-07 下午6 43 10" src="https://user-images.githubusercontent.com/72157947/117438564-4be7b980-af64-11eb-8c93-cf989c015f8b.png">

**Comment by code007-1** [comment]
it tried it, and it is showing 11 in Console Window

**Comment by UnitySuperEditor** [comment]
[SuperEditor.dll.zip](https://github.com/UnitySuperEditor/SuperEditor/files/6445194/SuperEditor.dll.zip)
Try this DLL and I guess it should work
Just replace the original supereditor.dll,This time I used all the Unity internal methods and deleted the Microsoft methods
<img width="275" alt="屏幕快照 2021-05-08 下午12 06 27" src="https://user-images.githubusercontent.com/72157947/117526375-47201580-aff7-11eb-97a5-5d378e58e81b.png">

**Comment by code007-1** [comment]
Finally, with new dll file it is working fine as expected, Thanks for solving this issue :)
